azurerm/helpers/azure/resource_group.go

```diff
@@ -27,6 +27,16 @@ func SchemaResourceGroupNameDeprecated() *schema.Schema {
 	}
 }
 
+func SchemaResourceGroupNameForceNewDeprecated() *schema.Schema {
+	return &schema.Schema{
+		Type:         schema.TypeString,
+		Optional:     true,
+		ForceNew:     true,
```

Comment:
This function "SchemaResourceGroupNameForceNewDeprecated" is only for lb_backend_address_pool_resource.go since [this resource](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/internal/services/network/lb_backend_address_pool_resource.go#L23) doesn't include "Update" in schema. So I have to keep forcenew here for this resource.

Reply:
I think you are going to have to add an update function, and then make everything but that property force new then.